recompose-idea/src/main/kotlin/recompose/plugin/copypaste/KotlinPasteFromXmlDialog.kt

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2010-2015 JetBrains s.r.o.
```

Comment:
Is this borrowed from the Kotlin plugin code?

Reply:
Yes, i looked how the dialog from the KotlintoJava Converter is doing it and modified it. I changed the copyright notice